module/canvas/grid.mjs

```diff
@@ -12,8 +12,8 @@ export function measureDistances(segments, options={}) {
     let r = s.ray;
 
     // Determine the total distance traveled
-    let nx = Math.abs(Math.ceil(r.dx / d.size));
-    let ny = Math.abs(Math.ceil(r.dy / d.size));
+    let nx = Math.ceil(Math.abs(r.dx / d.size));
+    let ny = Math.ceil(Math.abs(r.dy / d.size));
```

Comment:
We have the same issue in core. But I guess it doesn't matter, because the segments passed by the `Ruler` always start and end in grid space centers. Either way it wouldn't give the right results if the segments didn't start and end in grid space centers. We should get the grid coordinates of the grid spaces that contain the origin and target of the rays and calculate the distance this way.